recipes/das_node.rb

```diff
@@ -0,0 +1,485 @@
+#rows_path = "#{domains_dir}/post.sql"
+
+case node['platform']
+when "rhel"
+  package "openssh-clients"
+end
+
+package "openssh-server"
+public_ip=my_public_ip()
+payara_config = "hopsworks-config"
+deployment_group = "hopsworks-dg"
+local_instance = "instance0"
+service_name="glassfish-instance"
+
+domain_name= node['hopsworks']['domain_name']
+domains_dir = node['glassfish']['domains_dir']
+admin_port = node['hopsworks']['admin']['port']
+username=node['hopsworks']['admin']['user']
+password=node['hopsworks']['admin']['password']
+
+config_nodes= node['hopsworks'].attribute?('config_node')? private_recipe_ips('hopsworks', 'config_node') : []
+
+current_version = node['hopsworks']['current_version']
+
+asadmin = "#{node['glassfish']['base_dir']}/versions/current/bin/asadmin"
+password_file = "#{domains_dir}/#{domain_name}_admin_passwd"
```

Comment:
If it's not too bothersome can you create helper functions to construct them? I've seen them in many places

Reply:
I thought about it too but I do not think it will be that useful.